swagger.yaml

```diff
@@ -259,6 +259,12 @@ paths:
         required: false
         schema:
           type: string
+      - name: status
+        in: query
+        description: Status of device being '0' for off or '1' for on
```

Comment:
I'm not sure this provides enough info about what this does. How about something like "To return all devices, omit this query parameter. Specify '0' to query for disconnected devices or specify '1' for connected devices. 

We'll probably need to update the description since it specifically says "Regardless of connected status"